### ISSUE TITLE
Missing input normalization ?

### ISSUE THREAD

**Issue by mazzma12** [developmentseed/fastai-serving#1]
Hi, 
I might be wrong, but I don't see the place were the input are normalized according to the mean and std of the training set. `pred_batch` expects tensor out of dataloader, hence already normalized I would say

**Comment by drewbo** [comment]
@mazzma12 it isn't abstracted correctly right now, but I'm handing normalization in [this line](https://github.com/developmentseed/fastai-serving/blob/master/src/server.py#L50), assuming it is the first transformation. Only the first transform is applied right now (which should be updated) and this line could be further improved to operate over a tensor instead of iterating over a list. Let me know if you'd like help with a PR to this effect

**Comment by mazzma12** [comment]
I understand, but it's not obvious that the first transform is the normalization. For example when you call `get_transforms()` for generic transformation you end up with a crop pad transform as the only valid transform
```
RandTransform(tfm=TfmCrop (crop_pad), kwargs={}, p=1.0, resolved={}, do_run=True, is_random=True, use_on_y=True)
```

**Comment by drewbo** [comment]
Totally agree, I agree it's a bug just mentioning that it's possible to normalize through the current implementation if it's the only transform (because generally the other transforms aren't applied at inference time)
